pkg/ibmcsidriver/controller.go

```diff
@@ -110,6 +111,11 @@ func (csiCS *CSIControllerServer) CreateVolume(ctx context.Context, req *csi.Cre
 		return nil, commonError.GetCSIError(ctxLogger, commonError.InvalidParameters, requestID, err)
 	}
 
+	if requestedVolume.Profile != nil && requestedVolume.Profile.Name == RFSProfile && !csiCS.Driver.rfsEnabled {
```

Comment:
can you validate this condition with dp2 profile, looks it will also print that user need to get this enabled from vpc